index.js

```diff
@@ -1,89 +1,136 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+
+
+console.log("%cQuestion 1", "color: red") 
+
+var counter = 1
```

Comment:
You will want to avoid variable declarations with `var`. It will only be used in legacy code moving forward because it can cause problems relating to scope. Here would be a good place to use `let`